packages/grid/_modules_/grid/hooks/features/columns/gridColumnsSelector.ts

```diff
@@ -22,11 +22,11 @@ export const gridColumnsMetaSelector = createSelector(
     const positions: number[] = [];
 
     const totalWidth = visibleColumns.reduce((acc, curCol) => {
-      positions.push(acc);
+      positions.push(Math.round(acc));
```

Comment:
It seems that #1775 is already fixed in this PR without this change. Now the column widths will still be rendered with a non-integer value but `positions` and `totalWidth` will be integers. I remember that the idea of allowing non-integers was to not leave any blank space due the difference between the container width and the sum of the columns.

Reply:
I'm sceptical. I had the bug with this PR before but as you said if I remove the `Math.round` it still works.

For me the problem with the rounding was when we started rounding several columns, because at the rend, the rounded total was several pixel off the real total. 
But with this `Math.round` it should not be a problem.

I removed the `Math.round`. We will need to stay vigilant on that one.